pkg/app/app.go

```diff
@@ -1,53 +1,66 @@
 package app
 
 import (
+	"fmt"
 	"time"
 
-	"github.com/pkg/errors"
 	"github.com/replicatedhq/kots/pkg/app/types"
 	"github.com/replicatedhq/kots/pkg/persistence"
+	"github.com/rqlite/gorqlite"
 )
 
 // SetLastUpdateAtTime sets the time that the client last checked for an update to now
 func SetLastUpdateAtTime(appID string, t time.Time) error {
 	db := persistence.MustGetDBSession()
-	query := `update app set last_update_check_at = $1 where id = $2`
-	_, err := db.Exec(query, t, appID)
+	query := `update app set last_update_check_at = ? where id = ?`
+	wr, err := db.WriteOneParameterized(gorqlite.ParameterizedStatement{
+		Query:     query,
+		Arguments: []interface{}{t.Unix(), appID},
+	})
 	if err != nil {
-		return errors.Wrap(err, "failed to update last_update_check_at")
+		return fmt.Errorf("failed to update last_update_check_at: %v: %v", err, wr.Err)
```

Comment:
How does RQLite API work:   If `err != nil` then `wr.Err` will also be set?  And will they not contain the same message?

How about
```
errors.Wrapf(err, "failed to update last_update_check_at: %v", wr.Err)
```

Reply:
This one is odd and that's the best format I was able to come up with. Basically, if there is an error, both `err` and `wr.Err` will be set, but they'll contain different messages, like:

- `err` would be `there were 2 statement errors`
- `wr.Err` would be the actual error, e.g. `syntax error, unexpected )` or something like that.

i tried doing it the way you're suggesting, but the error message ended up weird, like:
```
failed to update: syntax error, unexpected ), there were 2 statement errors
```

but the way i currently have it, it would look a bit better, like:
```
failed to update: there were 2 statement errors: syntax error, unexpected )
```